src/lib/tabs/tab-header.spec.ts

```diff
@@ -382,6 +382,23 @@ describe('MatTabHeader', () => {
     });
 
   });
+
+    it('scroll to last tab', () => {
```

Comment:
I'm not sure that this tests the issue that was reported in #12889. It looks like it only tests that `_scrollToLabel` works.